doc/source/examples/cd_clf_adult.ipynb

```diff
@@ -258,7 +258,8 @@
      "output_type": "stream",
```

Comment:
<p>I'm curious where does the 63% figure come from? Probably a "citation needed" moment here :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
From [Bostrom](https://people.dsv.su.se/~henke/papers/bostrom08b.pdf) quotes this figure: 

_"Since on average about 0.632 of the training examples are used to grow
a single tree in the forest, it means that each example will
be out-of-bag for a little more than a third of the trees in the
forest."_

Reply:
Hmm, is this something that's configurable though? In any case, let's include the citation.

Reply:
Don't hold me to this, but I believe it's set by the `max_samples` in scikit-learn. Agree it should probably be clarified and cited here.

Reply:
This is a consequence of the bootstrapping. Each tree is trained on a dataset obtained by sampling with replacement from the original dataset. Let us consider that the original dataset has `N ` instances. When sampling with replacement, a training instance has a probability of being missed equal to `1-1/N` each time we draw a sample from the original dataset. Since we sample `N` times, the probability of an instance to be missed (i.e., never selected) from the bootstrap dataset is equal to `(1 - 1/N)^N` which converges to `1/e` as `N` goes to infinity. Thus, the expected number of missed samples is equal to `N/e`, which implies that the expected number of unique samples is `N - N/e = N(1 - 1/e)`. Now, `1 - 1/e` is approximately 0.63.

Reply:
I think reference to the paper is enough.